knora/dsplib/utils/excel_to_json_lists.py

```diff
@@ -106,15 +109,16 @@ def _get_values_from_excel(
         # previous values
         preval.append(base_file_ws.cell(column=col-1, row=row).value.strip())
 
-    while cell.value:
+    while cell.value and regex.search(r"\p{L}", cell.value, flags=re.UNICODE):
```

Comment:
Adds robustness. If many people work with an Excel file over some time, you inevitably get cells with whitespaces or other characters in them. So it is important to check if the cell contains a real string or just rubbish.